Tests/SentryTests/SentryViewHierarchyTests.swift

```diff
@@ -37,7 +37,7 @@ class SentryViewHierarchyTests: XCTestCase {
 
     override func tearDown() {
         super.tearDown()
-        clearTestState()
+        SentryDependencyContainer.reset()
```

Comment:
m: What is the impact of this change? Will we need to be aware of which teardown api to use for every test now?